src/app/screens/book-detail/book-detail.component.html

```diff
@@ -0,0 +1,24 @@
+<body class="detail">
+  <a class="back-arrow" [routerLink]="['/books']" >Atrás</a>
+  <div class="container">
+    <h2 class="bold-title">{{book.title}}</h2>
+    <span class="base-span subtitle">({{book.genre}})</span>
+    <div class="container-item" [ngClass]="{ badge: book.author == 'Chi Koepp' }">
```

Comment:
{ badge: book.author == 'Chi Koepp' } why this?

Reply:
In this case, I choose `Chi Koepp`

<img width="535" alt="Screen Shot 2020-05-05 at 17 24 38" src="https://user-images.githubusercontent.com/50454914/81121870-54428a00-8ef5-11ea-8321-a0b126352821.png">

Reply:
ok so store the value in a variable